app/components/works/show/authors_contributors_component.html.erb

```diff
@@ -5,13 +5,16 @@
     <tbody>
     <% participants.each do |participant| %>
       <tr>
-        <% if participant.person? %>
-          <td style="width: 50%">
+        <td style="width: 50%">
+          <% if participant.person? %>
             <%= participant.first_name %> <%= participant.last_name %>
-          </td>
-        <% else %>
-          <td style="width: 50%"><%= participant.full_name %></td>
-        <% end %>
+          <% else %>
+            <%= participant.full_name %>
+          <% end %>
+          <% if participant.affiliations.any? %>
+            (<%= participant.affiliations.map { |aff| aff.department ? "#{aff.label}, #{aff.department}" : aff.label }.join("; ") %>)
```

Comment:
I wonder if we should consider adding this affiliation display logic to the model. So here we'd have:

```suggestion
            (<%= participant.affiliations.map(&:to_s).join("; ") %>)
```

And in the `Affiliation` model we'd have something like:

```ruby
class Affiliation < ApplicationRecord
  # ...
  def to_s
    return label if department.blank?

    "#{label}, #{department}"
  end
  # or the equivalent:
  def to_s
    [label, department].compact_blank.join(", ")
  end
  # ...
end
```

Reply:
Much cleaner. Fixed.

Reply:
Thanks!